app/models/prospect.rb

```diff
@@ -1,2 +1,6 @@
 class Prospect < ApplicationRecord
+  validates :name, presence: true
+  validates :description, presence: true
+  validates :target, inclusion: {in: [true, false]}
```

Comment:
I wanted to have `presence: true` so `nil` is not a valid value. Unfortunately, it is not possible if you want to create an object with `false`.